File: app/src/main/java/com/master/kit/testcase/NewInstanceActivity.java
package com.master.kit.testcase;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.master.kit.R;
import tf.oof.com.service.base.BaseSlidingActivity;

import butterknife.BindView;
import butterknife.ButterKnife;

public class NewInstanceActivity extends BaseSlidingActivity {

    @BindView(R.id.click)
    Button click;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_new_instance);
        ButterKnife.bind(this);
        click.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(MainActivity.class);
            }
        });
    }
}

File: app/src/main/java/com/master/kit/testcase/NewInstanceActivity.kt
package com.master.kit.testcase

import android.os.Bundle
import butterknife.ButterKnife
import com.master.kit.R
import com.master.kit.activity.MainActivity
import com.ooftf.service.base.BaseSlidingActivity
import kotlinx.android.synthetic.main.activity_new_instance.*

class NewInstanceActivity : BaseSlidingActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_new_instance)
        ButterKnife.bind(this)
        click.setOnClickListener { startActivity(MainActivity::class.java) }
    }
}
